marklogic-data-hub-central/ui/src/components/entities/mapping/entity-map-table/entity-map-table.tsx

```diff
@@ -364,25 +369,118 @@ const EntityMapTable: React.FC<Props> = (props) => {
     }
   });
 
+  const getFilteredData = (value, tree) => {
```

Comment:
We have an object passed in as "tree", deep-copied to "filteredArray", and then used in the parser function as the arg "data". Does it make sense to have these names more consistent? Are we dealing with an array or a tree?

Reply:
Fixed. Thanks for reviewing.